src/lib/ngsi/Metadata.cpp

```diff
@@ -284,20 +270,6 @@ std::string Metadata::render(bool comma)
     out += JSON_STR("value") + ":" + JSON_STR("unknown json type");
   }
 
-  if ((valueType == orion::ValueTypeNumber) || (valueType == orion::ValueTypeBoolean) || (valueType == orion::ValueTypeNull))
-  {
-    //
-    // Adding newline for the types that do not use the valueTag() function
-    //
-    // FIXME: This might destroy V2 rendering
-    //   This newline is only desired for V1 requests and as this function hasn't that knowledge, we 'hardcode'
-    //   V1 behavior here, as V2 requests should use toJson and not render().
-    //   So, if V2 rendering is destroyed by this modification, it is only because the V2 rendering is using
-    //   a method that it SHOULD NOT USE !
-    //
-    out += "\n";
```

Comment:
This is the "\n" that I mention in .test comment was removed.

NTC (informative)